ui/src/hooks/useFileManage.ts

```diff
@@ -251,10 +251,6 @@ const initSocketEvent = (socket: WebSocket, t: any) => {
 
         if (message.cmd === 'upload' && message.data !== 'ok') {
           fileManageStore.setReceived(true);
-
-          globalTipsMessage.success(t('UploadSuccess'));
-
-          mittBus.emit('reload-table');
         }
 
         if (message.cmd === 'download' && message.data) {
```

Comment:
There is no apparent issue with this code. However, it is recommended to include comments explaining each variable and their purpose, as it can make the code more readable and maintainable.

For performance optimization, consider implementing lazy-loading of files that you know won't be used right away or using an asynchronous way to handle data received from socket.io events instead of synchronous operations like `setTimeout`, which could cause race conditions.

Lastly, ensuring that all methods throw meaningful exceptions when errors occur will enhance robustness of the application.